pmm/v3/pmm3-ovf-staging-start.groovy

```diff
@@ -112,14 +113,17 @@ pipeline {
                         for p in \$(seq 0 30); do
                             VBoxManage modifyvm ${VM_NAME} --natpf1 "guestexporters\$p,tcp,,4200\$p,,4200\$p"
                         done
+                        VBoxManage modifyvm ${VM_NAME} --vrde on
+                        VBoxManage modifyvm ${VM_NAME} --vrdeport 5000
                         VBoxManage startvm --type headless ${VM_NAME}
                         cat /tmp/${VM_NAME}-console.log
                         timeout 100 bash -c 'until curl --insecure -LI https://${IP}; do sleep 5; done' || true
+                        echo "${DEFAULT_SSH_KEYS}" >> /root/.ssh/authorized_keys
```

Comment:
Can we add an `SSH_KEY` parameter to this pipeline so that the job owner also gets ssh access to the instance? Similar to our `pmm3-aws-staging-start` pipeline.

Reply:
Good idea, will add